test/Interop/Cxx/class/access-specifiers-typechecker.swift

```diff
@@ -16,28 +15,33 @@ v.publicMemberFunc()
 var publicTypedefVar: PublicPrivate.PublicTypedef
 var publicStructVar: PublicPrivate.PublicStruct
 var publicEnumVar: PublicPrivate.PublicEnum
-// TODO: These enum values don't yet appear to be imported correctly into the
-// scope of PublicPrivate yet. Once they are, verify that they are accessible.
-// print(PublicPrivate.PublicEnumValue1)
-// print(PublicPrivate.PublicAnonymousEnumValue1)
 var publicClosedEnumVar: PublicPrivate.PublicClosedEnum
 var publicOpenEnumVar: PublicPrivate.PublicOpenEnum
 var publicFlagEnumVar: PublicPrivate.PublicFlagEnum
 
+// TODO: nested enum members aren't being imported correctly yet. rdar://143859248
```

Comment:
I think we try to not reference rdars in the source code as open source contributors have no access to them. We are OK to reference them in the commit messages though.